### DISCUSSION TITLE
What board do you use?

### DISCUSSION THREAD

**Discussion by renatoianhez** [schreibfaul1/ESP32-audioI2S#853] [General]
To build my radios that use this great library, I have used several Lolin32 Lite boards, with the addition of PSRAM via piggybacking. Later I started using ESP32 WROVER boards, which have native PSRAM and cost twice as much as Lolin32 boards, but are more reliable than adapting the memory. Now the import cost has gone up a lot, making it unfeasible (they created new taxes for imports in small quantities to Brazil). My question is: what board do you use that supports the library? Is there a cheaper alternative?

**Comment by jonkstro** [comment]
Im trying to use the Lolin ESP32 but the IDE is not compiling, it says that the flash size doesnt support the example code 🫤
